core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -608,6 +608,18 @@ portfolio.pending.step2_seller.waitPayment.msg=The deposit transaction has at le
 portfolio.pending.step2_seller.warn=The BTC buyer still has not done the {0} payment.\nYou need to wait until they have started the payment.\nIf the trade has not been completed on {1} the arbitrator will investigate.
 portfolio.pending.step2_seller.openForDispute=The BTC buyer has not started his payment!\nThe max. allowed period for the trade has elapsed.\nYou can wait longer and give the trading peer more time or contact the arbitrator for opening a dispute.
 
+tradeChat.chatWindowTitle=Chat window for trade with ID ''{0}''
+tradeChat.openChat=Open chat window
+tradeChat.rules=You can communicate with your trade peer to resolve potential problems with this trade. It is not mandatory to reply in \
```

Comment:
```
tradeChat.rules=Here you can communicate with your trading peer to resolve potential problems with this trade.\n\n\
It is not mandatory to reply to any messages.\n\nIf a trader violates any of the rules below, open a dispute with \
'Ctrl/Cmd + o' and report it to the arbitrator.\n\n\
```

Main updates here: 

- put "it is not mandatory to reply to any messages" on its own line so it is crystal clear
- Add 'Cntrl/Cmd' for non-Mac users